core/src/test/java/org/infinispan/interceptors/impl/MarshalledValueInterceptorTest.java

```diff
@@ -1,4 +1,4 @@
-package org.infinispan.interceptors;
+package org.infinispan.interceptors.impl;
```

Comment:
True, it could have stayed in the old package. The full history is that at some point I had copied all the interceptors for backwards compatibility, and I copied this test as well. Then we changed the major and I was able to remove the old interceptors, and I saw no reason to move the "new" test.